integration-tests/src/server.js

```diff
@@ -20,26 +21,44 @@ const app = express()
 const server = http.createServer(app)
 server.setTimeout(serverTimeout)
 
+const firestore = new Firestore()
+
 const testTimeout = async timeout => {
   await delay(timeout)
   throw new Error(`Test timeout after ${timeout}ms`)
 }
 
+const reportTest = (testGroup, startTime, host, testName, errorMessage) => {
```

Comment:
TOL: Rather than passing all these parameters, it might be cleaner to define this helper inside `runTest`, and just reference everything except `errorMessage` from the lexical scope.